framework/helpers/BaseConsole.php

```diff
@@ -340,6 +340,93 @@ public static function ansiStrlen($string)
         return mb_strlen(static::stripAnsiFormat($string));
     }
 
+    //
+    // Exit code constant names, values, and descriptions are taken from
+    // OpenBSD's sysexits(3) man page http://man.openbsd.org/sysexits
+    //
+    // The command completed successfully.
+    const EX_OK = 0;
```

Comment:
@samdark That arguments holds for most function in the `ConsoleHelper` as well ;-)

What if I log these exit codes and want to display them in a web page? It'd be more logical to do inflection on the helper than the controller.
Also a controller should be skinny in MVC, adding a list of constants does not seem obvious to me.
Also, a console application can use controllers that are not descendants of the console controller, but of the base controller.